daemon/daemon_main.go

```diff
@@ -525,6 +527,8 @@ func init() {
 	flags.BoolVar(&enableDNSPoller,
 		"tofqdns-enable-poller", false, "Enable proactive polling of DNS names in toFQDNs.matchName rules.")
 
+	flags.IntVar(&rejectDNSResponseCode, "tofqdns-dns-response-code", dnsproxy.ProxyDenyWithRefused, "DNS response code for rejecting DNS request")
```

Comment:
I think it would be a lot more user friendly to allow specifying this as string and do the translation to the opcode in `SetRejectReply()`. While you are at it, please define the option name in `pkg/option` and add a `BindEnv()` so users can use the ConfigMap easily.